dev/com.ibm.ws.persistence-1.0/com.ibm.ws.persistence-1.0.feature

```diff
@@ -4,7 +4,7 @@ visibility=private
 IBM-API-Package: com.ibm.websphere.persistence.mbean; type="ibm-api"
 -features=com.ibm.websphere.appserver.jndi-1.0, \
  com.ibm.websphere.appserver.containerServices-1.0, \
- com.ibm.websphere.appserver.javax.annotation-1.2; apiJar=false, \
+ com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3; apiJar=false, \
```

Comment:
anything that includes persistence-1.0 will need to propagate this line in order to disambiguate feature toleration -- this can be tested out locally by loading up public features in a server.xml